src/rng.cpp

```diff
@@ -113,6 +113,23 @@ int djb2_hash( const unsigned char *input )
     return hash;
 }
 
+std::vector<int> rng_sequence( size_t count, int lo, int hi, int seed )
+{
+    static std::uniform_int_distribution<int> rng_int_dist;
```

Comment:
This should probably not be static as that could leak data from one invocation to the next one. (`uniform_int_distribution::operator()` is not `const`, it may change the state of the distribution.)

Or call `reset` before using the distribution again.

Reply:
Was like 90% sure it's const, but turns out it was [MSVC quirk](https://developercommunity.visualstudio.com/t/operator-of-uniform-int-distribution-is-a-const-me/740558), thanks